database/dbService.js

```diff
@@ -62,6 +62,8 @@ module.exports = {
 
     /* Retrieves a User model by ID */
     getUserByID: function(ID, callback) {
+
+        console.log(ID);
         this.tryConnect().getConnection(function(err, con) {
```

Comment:
Assuming this will be deleted when you're done?

Reply:
I didn't code that. Should I delete it?

Reply:
Yeah